:bug: Fix url to aturi parser

foysalit opened the pull request:
This PR ensures that at-uri is only built with a DID and not a handle from command palette input.

devinivy commented on lib/client.ts:
```diff
@@ -133,7 +133,10 @@ class ClientManager extends EventTarget {
-      ozoneDid || process.env.NEXT_PUBLIC_OZONE_SERVICE_DID || undefined
+      ozoneDid ||
+      process.env.NEXT_PUBLIC_OZONE_SERVICE_DID ||
+      'did:plc:kqg2icildynagykzlxehkkaw' ||
```
I think this may be left around from dev.

foysalit replied:
yep. cleaned up!

devinivy reviewed (commented):
Looks good!

foysalit commented:
> Looks good! (Any idea what the failing E2E tests are about?)

yep, added some new tests around the changes and the new one was failing. fixed those up just now.
